Core/Object Arts/Dolphin/ActiveX/OCX/AXControlSite.cls

```diff
@@ -795,6 +795,29 @@ state
 	sequence list addAll: controlState afterIndex: 1.
 	^sequence!
 
+storeOn: aStream indent: anInteger variables: aDictionary
+
+	| default name |
+	name := self storeNameIn: aDictionary.
+	aStream display: name; cr.
+	anInteger + 1 timesRepeat: [aStream tab].
+	[
+		default := (self class progId: self progId) create; show.
+		aStream
+			nextPutAll: ' progId: ';
```

Comment:
A general approach we follow when "printing" code to a stream, is to display the actual selector symbol. This means that subsequently this will be found when browsing for references, and would mean that the code would participate in any refactoring operation such as a rename that changed the selector. This is slightly more awkward than using strings, but is worth it for the reference browsing alone.